01-Login/src/app/app-routing.module.ts

```diff
@@ -20,14 +16,14 @@ export const routes: Routes = [
     canActivate: [AuthGuard]
   },
   {
-    path: 'login',
```

Comment:
why we don't have this path anymore?

Reply:
This path wasn't being used for anything / wasn't called anywhere in the app, as far as I could tell. It didn't have a component associated with it and was never requested